recipe/build.sh

```diff
@@ -164,12 +189,19 @@ elif [[ ${cuda_compiler_version} != "None" ]]; then
             echo "unknown CUDA arch, edit build.sh"
             exit 1
     esac
+    # Warning from pytorch v1.12.1: In the future we will require one to
+    # explicitly pass TORCH_CUDA_ARCH_LIST to cmake instead of implicitly
+    # setting it as an env variable.
+    #
+    # See:
+    # https://pytorch.org/docs/stable/cpp_extension.html (Compute capabilities)
+    # https://github.com/pytorch/pytorch/blob/main/.ci/manywheel/build_cuda.sh
     case ${cuda_compiler_version} in
         12.6)
             export TORCH_CUDA_ARCH_LIST="5.0;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0+PTX"
             ;;
```

Comment:
So what should we do differently to prepare for this?

Reply:
probably worth attending to this now, you're right, will have a look

Reply:
have handed the variable via CMAKE_ARGS

Reply:
I was seeing `unused variables: CMAKE_ARGS` in the config for some platforms, so I'm not confident this is working. Will add a note to address this at a later date unless you object. I tried a couple of ways to hand this directly to CMake but given we're using setup.py as an entry point it's nontrivial